src/Conversations/Conversation.php

```diff
@@ -179,6 +195,14 @@ public function hydrate(array $data, array $embedded = [])
         $this->setPreview($data['preview'] ?? null);
         $this->setMailboxId($data['mailboxId'] ?? null);
 
+        // Hydrating a Conversation from a webhook it'll have a Mailbox
+        if (isset($data['mailbox'])) {
+            $mailbox = new Mailbox();
+            $mailbox->hydrate($data['mailbox']);
+            $this->setMailbox($mailbox);
```

Comment:
I don't fully understand the context, but could `setMailbox` perhaps internally set the mailbox ID rather than having the two calls here?

Reply:
I updated the comment block surrounding this section of code to better explain why we're also setting the ID.  Can you check and see if that makes more sense now?